packages/@uppy/tus/src/index.js

```diff
@@ -278,15 +303,7 @@ module.exports = class Tus extends BasePlugin {
       this.uploaders[file.id] = upload
       this.uploaderEvents[file.id] = new EventTracker(this.uppy)
 
-      upload.findPreviousUploads().then((previousUploads) => {
-        const previousUpload = previousUploads[0]
-        if (previousUpload) {
-          this.uppy.log(`[Tus] Resuming upload of ${file.id} started at ${previousUpload.creationTime}`)
-          upload.resumeFromPreviousUpload(previousUpload)
-        }
-      })
-
-      let queuedRequest = this.requests.run(() => {
+      queueRequest = () => {
```

Comment:
The naming suggests this to be an object (with a `run` method?), not a function, but I realize the pattern is used all over the file, so feel free to ignore me. This would be a big refactoring to keep it consistent in the file.